mkosi/__init__.py

```diff
@@ -1545,6 +1545,46 @@ def build_initrd(context: Context) -> Path:
     return config.output_dir / config.output
 
 
+def identify_cpu(root: Path) -> tuple[Optional[Path], Optional[Path]]:
+    for entry in Path("/proc/cpuinfo").read_text().split("\n\n"):
```

Comment:
We should skip this if we're not building for x86

Reply:
I moved the skip into `build_microcode_initrd`